node/router/messages/src/block_request.rs

```diff
@@ -18,7 +18,7 @@ use snarkvm::prelude::{FromBytes, ToBytes};
 
 use std::borrow::Cow;
 
-#[derive(Copy, Clone, Debug, PartialEq, Eq, Hash)]
+#[derive(Copy, Clone, Debug, PartialEq, Eq, Hash, Default)]
```

Comment:
I would suggest not using `Default` here because this would technically be a malformed `BlockRequest` (which goes against the spirit of the purpose for `Default` in Rust)

Reply:
7e16a7fab